doc/source/data/inspecting-data.rst

```diff
@@ -0,0 +1,130 @@
+.. _inspecting-data:
+
+===============
+Inspecting data
+===============
+
```

Comment:
can you add a sentence or 2 about what this guide covers? Just consistent pattern with the other user guides.